demo.cast

```diff
@@ -0,0 +1,565 @@
+{"version": 2, "title": "Cloudsmith CLI", "width": 194, "height": 74, "timestamp": 1595593770, "env": {"SHELL": "/usr/local/bin/bash", "TERM": "xterm-256color"}}
```

Comment:
we can probably skip uploading this file; I'm not sure there is much we can do to reuse them other than to upload it again.